File: athrill-account-server/Libraries/Model/Basics/Teams/TeamCategoryMemberLinkModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AT.Model.Basics.Teams
{
    public class TeamCategoryMemberLinkModel : BaseModel
    {
        public int TeamCategoryMemberLinkId { get; set; }
        public int TeamCategoryId { get; set; }
        public int TeamMemberId { get; set; }
        public int TeamMemberOrientation { get; set; }

        #region Override

        public override int Id => TeamCategoryMemberLinkId;

        #endregion


        #region Links

        public TeamCategoryModel TeamCategory { get; set; }
        public TeamMemberModel TeamMember { get; set; }

        #endregion
    }
}

File: athrill-account-client/projects/at-models/src/lib/basics/teams/team-category-member-link.model.ts
import { BaseModel } from '../../at-base.model';
import { TeamCategoryModel } from './team-category.model';
import { TeamMemberModel } from './team-member.model';

export class TeamCategoryMemberLinkModel extends BaseModel{
    public TeamCategoryMemberLinkId: number;
    public TeamCategoryId: number;
    public TeamMemberId: number;
    public TeamMemberOrientation: number;

    public Id: number;

    public TeamCategory: TeamCategoryModel;
    public TeamMember: TeamMemberModel;

}
